task/04-arrays-tasks.js

```diff
@@ -338,7 +337,7 @@ function sortDigitNamesByNumericOrder(arr) {
  *   [ 1, 10, 100, 1000 ]  => 1111
  */
 function getItemsSum(arr) {
-  throw new Error('Not implemented');
+  return arr.reduce((a, item) => a + item, 0 );
```

Comment:
specifying the 3-rd argument to the `reduce` is not necessary: by default it will equal the first item of an array and the sum will work correctly.